src/AutoVizuA11y.tsx

```diff
@@ -262,40 +284,59 @@ const AutoVizuA11y = ({
 			});
 
 			//wipes the old tabindex present in the child components
-			levelChart(ref, true);
+			switchToChartLevel(ref, true);
 		}, 500);
 
 		//creates the ShortcutGuide
 		createShortcutGuide();
 	}, [ref]);
 
-	//sets the appropriate navigation keys in the ShortcutGuide
+	// sets the appropriate navigation keys in the ShortcutGuide
 	function handleNav(event: React.KeyboardEvent<HTMLDivElement>) {
-		levelNav(event, ref);
+		navigationKeyHandler(
```

Comment:
As a good measure, usually we try to at, at most, 3 params in a function definitions.
If we have more that 3, then we convert those 3 into a single object.

It makes code more readable and easier to maintain.